addons/binding/org.openhab.binding.seneye/ESH-INF/binding/binding.xml

```diff
@@ -6,6 +6,6 @@
 
 	<name>Seneye Binding</name>
 	<description>The seneye binding polls the seneye API for your aquarium readings.</description>
-	<author>Niko Tanghe</author>
+	<author>Niko Tanghe, Updated by Mick Ursell</author>
```

Comment:
I think in general this tag is intended for the original author of the binding or if you have made major changes to the binding. I've also only seen it used with only names in it. It is more common for small changes to add the @author tag in the class comment, something like `@author you name - Added additional channels`.